client/components/form.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import 'bootstrap/dist/css/bootstrap.css';
+
+const Form = () => {
+
+  const [name, setName] = useState('');
+  const [phoneNumber, setPhoneNumber] = useState('');
+  const [barberName, setBarberName] = useState('');
+
+  const submitForm = async event => {
+    event.preventDefault();
+    try {
+      const body = { name, phoneNumber, barberName };
+      // eslint-disable-next-line
+      const response = await fetch('http://localhost:3000/api/waitlist', {
+        method: 'POST',
+        headers: { 'Content-Type': 'application/json' },
+        body: JSON.stringify(body)
+      });
+      window.location = '/';
```

Comment:
This shouldn't be necessary. You don't need to reload the universe just to show an extra table row. The fetch-in-react-practice POST feature does what you need.